labgavin/src/component/dashboard-container/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import './_dashboard-container.scss';
+import {connect} from 'react-redux';
+import {
+  categoryCreate,
+  categoryUpdate,
+  categoryDelete,
+} from '../../action/category-actions';
+import CategoryForm from '../category-form';
+import CategoryItem from '../category-item';
+
+class DashboardContainer extends React.Component {
+  componentDidMount() {
+    console.log('__DASHBOARD__', this);
+    this.props.categoryCreate({title: 'Dining Out', budget: '500'});
+
+  }
+
+  render() {
+    return (
+      <main className="dashboard-container">
+        <h1>Dashboard</h1>
+        <h2>Total Monthly Budget: {this.props.totalMonthly}</h2>
+        <h2>Total remaining: {this.props.totalMonthly}</h2>
+
+        <CategoryForm
+          buttonText='Create'
```

Comment:
this component should support an optional category that will initialize the state of the form